scripts/utils/helper.js

```diff
@@ -0,0 +1,121 @@
+
+export function init(Model,Controller,View) {
+    const modal = new Model();
+    const view = new View();
+    const controller = new Controller({
+      modal,
+      view,
+    });
+    return controller;
+}
+
+export var createSubcategory= function(info){
+  var element= document.createElement("div");
+  var element_first_child=document.createElement("div");
+  var element_second_child=document.createElement("div");
+  var img_child=document.createElement("img");
+
+  element.setAttribute("category",info.category);
+  element.setAttribute("subcategory",info.subcategory);
+  element_first_child.className="aside-img";
+  element_second_child.className="aside-content";
+  element_second_child.innerText= info.subcategory;
+  img_child.src= info.img_src;
+  img_child.alt= info.img_alt;
+
+  element.appendChild(element_first_child);
+  element.appendChild(element_second_child);
+  element_first_child.appendChild(img_child);
+  
+  return element;
+} 
+
+export var createCard = function(info){
+  var element= document.createElement("div");
```

Comment:
SRP is not maintained in this function, please refactor

Reply:
converted this function and other that are big like this to back-tick /template string format